lib/ContextResolver.js

```diff
@@ -153,6 +153,9 @@ module.exports = class ContextResolver {
       context = remoteDoc.document || null;
       // parse string context as JSON
       if(_isString(context)) {
+        if (context.charCodeAt(0) === 0xFEFF) {
+          context = context.slice(1);
```

Comment:
This seems to be happening at the wrong layer. I would expect, if anything is to handle this, it would be a document loader that would strip it from the bytes it receives.